site/content/docs/5.3/forms/layout.md

```diff
@@ -217,6 +235,12 @@ As shown in the previous examples, our grid system allows you to place any numbe
 
 The example below uses a flexbox utility to vertically center the contents and changes `.col` to `.col-auto` so that your columns only take up as much space as needed. Put another way, the column sizes itself based on the contents.
 
+{{< design-callout-alert >}}
+These form variants, with an **horizontal layout** (i.e. labels not above the input fields), should not be used because they do not respect the Orange Design System specifications.
```

Comment:
Maybe recheck that each sentence is the same everywhere, this one is different of the 2 others in this file

Reply:
OK. To me, the one that is slightly different is the one juste before where the sentence "The input label should always be placed above the input field." remains. It should be removed there to be consistent.